### ISSUE TITLE
Multiple bar charts with multiple Y axes Not Working

### ISSUE THREAD

**Issue by plan2execute** [alexandersimoes/d3plus#560]
we are trying to implement bar graph with multiple y axes. Please see the below fiddle.

https://jsfiddle.net/pyc44ghr/4/

As you can see, we are able to view only two bars for the first value.
Is there a way to make it available for all values?

**Comment by davelandry** [comment]
This fix will be part of the next release. :beers:

**Comment by ChandrasekaranK** [comment]
Dave,
Where can i download the release with this fix? pl help. your help is greatly appreciated.
Rgds,
CHandra

**Comment by ChandrasekaranK** [comment]
Dave,

I went through github and tried building the latest from github using the commands given in the homepage to build the latest dev version of d3plus. when i used the latest dev built d3plus.full.min.js still the issue is there. pl guide us. when the saw the changes history i saw bar.coffee has been changed for this fix. but still the issue persists (after building d3plus with latest changes from github). your help is highly appreciated.
Rgds,
Chandra
